feat: CSP enhancements (#5516)
codeigniter4/CodeIgniter4: user_guide_src/source/general/common_functions.rst

    Returns the timezone the application has been set to display dates in.

.. php:function:: csp_script_nonce ()

    :returns: The CSP nonce attribute for script tag.
    :rtype: string

>     Returns the nonce attribute for script tag. That is string like ``nonce="Eskdikejidojdk978Ad8jf"``.

Reviewer: ```suggestion
    Returns the nonce attribute for a script tag. For example: ``nonce="Eskdikejidojdk978Ad8jf"``.
```

    See :ref:`content-security-policy`.

.. php:function:: csp_style_nonce ()

    :returns: The CSP nonce attribute for style tag.
    :rtype: string

>     Returns the nonce attribute for style tag. That is string like ``nonce="Eskdikejidojdk978Ad8jf"``.

Reviewer: ```suggestion
    Returns the nonce attribute for a style tag. For example: ``nonce="Eskdikejidojdk978Ad8jf"``.
```
